pkg/api/types.go

```diff
@@ -1544,6 +1544,16 @@ type NodeStatus struct {
 	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
 	// Set of ids/uuids to uniquely identify the node.
 	NodeInfo NodeSystemInfo `json:"nodeInfo,omitempty"`
+	// List of container images on this node
+	Images []ContainerImage `json:"images",omitempty`
+}
+
+// Describe a container image
+type ContainerImage struct {
+	// Names by which this image is known.
+	RepoTags []string `json:"repoTags"`
```

Comment:
`NodeStatus` will be frozen for a while. So evolving this struct is hard. We can start with a simple model that we know for sure is cross runtime compatible. 
I talked to @bgrant0607 about this a bit, and we decided to just rename `RepoTags` to `Names` since aliases are a common enough pattern for most package distribution systems.